tests/test_sessions_api.py

```diff
@@ -0,0 +1,112 @@
+from datetime import datetime, timedelta
+from functools import partial
+
+import pytz
+import simplejson as json
+
+from snuba import settings, state
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.storages import StorageKey
+from snuba.clusters.cluster import get_cluster
+from snuba.datasets.storages.factory import get_writable_storage
+from tests.base import BaseApiTest
+from snuba.processor import MAX_UINT32
+from snuba.utils.metrics.backends.dummy import DummyMetricsBackend
+
+
+class TestSessionsApi(BaseApiTest):
+    def setup_method(self, test_method):
+        super().setup_method(test_method)
+        self.app.post = partial(self.app.post, headers={"referer": "test"})
+
+        # values for test data
+        self.minutes = 180
+        self.skew = timedelta(minutes=self.minutes)
+        self.started = datetime.utcnow().replace(
+            minute=0, second=0, microsecond=0, tzinfo=pytz.utc
+        )
+
+        self.storage = get_writable_storage(StorageKey.SESSIONS_RAW)
+        self.generate_manual_session_events()
+
+    def teardown_method(self, test_method):
+        # Reset rate limits
+        state.delete_config("global_concurrent_limit")
+        state.delete_config("global_per_second_limit")
+        state.delete_config("project_concurrent_limit")
+        state.delete_config("project_concurrent_limit_1")
+        state.delete_config("project_per_second_limit")
+        state.delete_config("date_align_seconds")
```

Comment:
Why are these needed ?

Reply:
I just copy-pasted those from `test_api.py`/`test_transactions_api.py`, I removed them and tests seem to be happy.